4-star-nosed_mole/api_definitions/rest/ucs.yaml

```diff
@@ -66,6 +66,11 @@ paths:
             Exceptions by ID:
               - noFileAccess: The user is not registered as a Data Submitter
                 for the corresponding file.
+        "404":
```

Comment:
Good idea, I am not sure about the status code, though. I think typically the 404 is used when the requested URL does not exist. Maybe it is better to also move this to the 400-list?